### ISSUE TITLE
ctc training not support large batch size?

### ISSUE THREAD

**Issue by beyondboy** [facebookresearch/wav2letter#179]
when i set batch-size=16 the training does not converge(loss oscillating back and forth) no matter how i adjust the learning rate(0.05,0.1...0.8). but when i set batch-size back to 4, it's conveged normal. what possible reason for this? (all other config the  same as tutorial)
part of training log as following:
```
poch:1 loss:91.69394 | train-TER:132.33 | valid_1000-TER:100.00 
epoch:1 loss:89.60210 | train-TER:100.00 | valid_1000-TER:100.00 
epoch:1 loss:78.05183 | train-TER:100.00 | valid_1000-TER:100.00
epoch:1 loss:28.90485 | train-TER:100.00 | valid_1000-TER:100.00
epoch:1 loss:13.43071 | train-TER:100.00 | valid_1000-TER:100.00 
epoch:1 loss:16.36989 | train-TER:100.00 | valid_1000-TER:100.00 
epoch:1 loss:31.02535 | train-TER:100.00 | valid_1000-TER:100.00
epoch:1 loss:20.60933 | train-TER:100.00 | valid_1000-TER:100.00 
epoch:1 loss:25.24020 | train-TER:100.00 | valid_1000-TER:100.00
```
